Makefile

```diff
@@ -109,12 +110,17 @@ third-party-test-venv: FORCE
 third-party-chpldoc-venv: FORCE
 	cd third-party && $(MAKE) chpldoc-venv
 
+third-party-scspell-venv: FORCE
```

Comment:
This was my thinking as well. I don't think a developer tool like this needs to end up in the users' hands